examples/README.md

```diff
@@ -33,17 +56,4 @@ The [runs](./runs/) dir contains an example [TaskRun](../README.md#taskrun) and
 
 [run-kritis-test.yaml](./invocations/run-kritis-test.yaml) shows an example of how to manually run kritis unit test off your development branch.
 
-[kritis-pipeline-run.yaml](./invocations/kritis-pipeline-run.yaml) shows an example of what it would look like to invoke the [kritis example pipeline](#example-pipelines) manually and have it fail on the second task (building and pushing the image).
-
-## Example Pipelines
-
-Finally, we have 2 example [Pipelines](../README.md#pipeline) in [./pipelines](./pipelines)
-
-1. [Kritis](./pipelines/kritis.yaml): This exmaple demonstrates how to configure a pipeline which runs unit test, build an image, deploys it to test and then run integration tests. (This is the pipeline for [kritis](https://github.com/grafeas/kritis).)
-
-![Pipeline Configuration](./pipelines/kritis-pipeline.png)
-
-2. [Guestbook](./pipelines/guestbook.yaml): This is pipeline which is based on example application in [Kubernetes example Repo](https://github.com/kubernetes/examples/tree/master/guestbook)
-This pipeline demonstartes how to integrate frontend [guestbook app code](https://github.com/kubernetes/examples/tree/master/guestbook-go) with backed [redis-docker image](https://github.com/GoogleCloudPlatform/redis-docker/tree/master/4) provided by GCP.
-
-![Pipeline Configuration](./pipelines/guestbook-pipeline.png)
+[kritis-pipeline-run.yaml](./invocations/kritis-pipeline-run.yaml) shows an example of what it would look like to invoke the [kritis example pipeline](#example-pipelines) manually and have it fail on the second task (building and pushing the image).
```

Comment:
This is a little bit confusing to me. 
I am reading it as "User has configured this pipeline to fail at step 2" 
Instead, the pipeline run controller determined the task failed and updated the status right?

Reply:
good point! I've tried to make this more clear, PTAL